src/fprime_bootstrap/bootstrap_project.py

```diff
@@ -181,6 +181,16 @@ def setup_git_repo(project_path: Path, tag: str):
         )
         sys.exit(1)
 
+    # Initial commit
```

Comment:
Traditional git usage is to have an empty commit.  Perhaps we shoul:

1. Commit first (empty)
2. add (here already)
3. Commit with results of bootstrap  (here already)